bundles/net.tourbook/src/net/tourbook/map2/action/ActionExportMapViewClipboard.java

```diff
@@ -42,14 +43,20 @@ public ActionExportMapViewClipboard(final Map2View mapView) {
    @Override
    public void run() {
 
-      final Image mapViewImage = _map2View.getMapViewImage();
-
       final Clipboard clipboard = new Clipboard(Display.getCurrent());
 
-      final Object imageTransfer = UI.IS_LINUX ? PngTransfer.getInstance() : ImageTransfer.getInstance();
-
-      clipboard.setContents(new Object[] { mapViewImage.getImageData() },
-            new Transfer[] { (ImageTransfer) imageTransfer });
+      final Image mapViewImage = _map2View.getMapViewImage();
+      final ImageData imageData = mapViewImage.getImageData();
+
+      if (UI.IS_LINUX) {
+         final PngTransfer imageTransfer = PngTransfer.getInstance();
+         clipboard.setContents(new Object[] { imageData },
+               new Transfer[] { imageTransfer });
+      } else {
+         final ImageTransfer imageTransfer = ImageTransfer.getInstance();
+         clipboard.setContents(new Object[] { imageData },
+               new Transfer[] { imageTransfer });
```

Comment:
The map is a little bit empty when pasting into gimp or inkscape

![mt-copy-map-to-clipboard](https://user-images.githubusercontent.com/1283445/105626124-3c69b800-5e2e-11eb-88f3-ce3b453bc4a7.png)

Reply:
I used MS Paint and it worked, can you try and tell me? 
There might be a transparent layer issue with the set code

Reply:
This is what I see in Windows:
- When pasted in MS Paint, it works fine
- When pasted in GIMP, it shows a transparent map
![image](https://user-images.githubusercontent.com/25235665/105661937-dd3a9080-5e8b-11eb-88b6-bb948550b850.png)

Reply:
* MS Paint is OK
* Inkscape 
 ** is OK when using `net.tourbook.map2.view.PngTransfer`
** is not OK when using `org.eclipse.swt.dnd.ImageTransfer`

FYI: `org.eclipse.swt.graphics.ImageData.transparentPixel` is used in the 2D map

![mt-map-image-clipboard](https://user-images.githubusercontent.com/1283445/105709967-c0976a80-5f16-11eb-86a5-95870fbf8f8a.png)

Reply:
> 
> 
>     * MS Paint is OK
> 
>     * Inkscape
>       ** is OK when using `net.tourbook.map2.view.PngTransfer`
>       ** is not OK when using `org.eclipse.swt.dnd.ImageTransfer`

Should we then ask the user to only use MS Paint ? That seems a bit restrictive.
I found [this bug](https://bugs.eclipse.org/bugs/show_bug.cgi?id=299138) that seems similar but the solution uses awt.

> 
> FYI: `org.eclipse.swt.graphics.ImageData.transparentPixel` is used in the 2D map
> 
I tried but it seems that the transparentPixel is not used
![image](https://user-images.githubusercontent.com/25235665/105767841-78e30400-5f53-11eb-81c6-cedd85febcb5.png)

Reply:
After more investigation, I have found a fix for this issue 😄 
Now, exporting the map as an image to the clipboard it works both on Linux and Windows.
I have created a PR

![image](https://user-images.githubusercontent.com/25235665/105772381-50411700-5f1f-11eb-99b1-84c0ba1b227b.png)

Reply:
🥇 

I played a little bit with the transparent pixel but was not successful

Reply:
For now it works good but maybe with time, a better solution will be found.